csharp/Facebook.CSSLayout/CSSLayoutContext.cs

```diff
@@ -0,0 +1,17 @@
+﻿/**
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+namespace Facebook.CSSLayout
+{
+    public class CSSLayoutContext
+    {
+        public MeasureOutput measureOutput { get; } = new MeasureOutput();
```

Comment:
Uppercase first character of property name (`measureOutput` -> `MeasureOutput`)?